core/src/com/cloud/agent/api/SecurityGroupRulesCmd.java

```diff
@@ -35,7 +35,7 @@
 public class SecurityGroupRulesCmd extends Command {
     private static final String CIDR_LENGTH_SEPARATOR = "/";
     private static final char RULE_TARGET_SEPARATOR = ',';
-    private static final char RULE_COMMAND_SEPARATOR = ':';
+    private static final char RULE_COMMAND_SEPARATOR = ';';
```

Comment:
Not that I'm sure of. I wouldn't know why as this is all run locally on the HV in KVM's case